documentation/SA1011.md

```diff
@@ -25,7 +25,12 @@ A violation of this rule occurs when the spacing around a closing square bracket
 
 A closing square bracket must never be preceded by whitespace, unless it is the first character on the line.
 
-A closing square bracket must be followed by whitespace, unless it is the last character on the line, it is followed by a closing bracket or an opening parenthesis, it is followed by a comma or semicolon, or it is followed by certain types of operator symbols.
+A closing square bracket must be followed by whitespace unless:
+* It is the last character on the line
+* It is followed by a closing bracket or an opening parenthesis
+* It is followed by a comma or semicolon
+* It is followed by a alignment component or format string component. For example: `$"{x[i],3}"` or `$"{x[i]:C}"`
```

Comment:
I would prefer `a string interpolation alignment component or string interpolation formatting component` instead of `a alignment component or format string component`